examples/recording_service/service_admin/cpp/README.rst

```diff
@@ -0,0 +1,143 @@
+  from the running service instance.
+
+- ``USER_QOS_PROFILES.xml``: this file defines DDS XML QoS profiles to be used
+  by the application and by *Recording Service*.
+
+- ``recorder_remote_admin.xml``: this is a configuration file for Recording
+  Service that enables remote administration, and uses the QoS profiles defined
+  in the XML QoS profiles file defined above. By using the same QoS profiles in
+  both the application and the *Recording Service* we ensure the QoS matching 
+  and the reliability of the communication.
+
+Building the Example
+====================
+
+In order to build this example, you need to provide the following variables to
+``CMake``:
+
+- ``CONNEXTDDS_DIR``: pointing to the installation of RTI Connext DDS to be 
+  used.
+
+- ``CONNEXTDDS_ARCH``: the RTI Connext DDS Target architecture to be used in 
+  your system.
+
+Build the example code by running the following command:
+
+.. code-block:: text
+
+    mkdir build
+    cd build
+    cmake -DCONNEXTDDS_DIR=<connext dir> -DCONNEXTDDS_ARCH=<connext architecture> ..
+    cmake --build .
+
+This will produce a binary directory (*build*) where the ``Requester`` 
+application can be found. The XML files in the source directory are also copied
+over to this binary directory so that |RS| can be run directly from this 
+directory as well.
+
+**Note**: in order to build, ``CMake`` will invoke *RTI DDS Code Generator* on
+the ``ServiceCommon.idl``, ``ServiceAdmin.idl`` and ``RecordingServiceTypes.idl``
+files that can be found in ``$(CONNEXTDDS_DIR)/resource/idl`` directory. These
+files define the communication interface for the remote administration platform
+and any |RS|-specific types (``RecordingServiceTypes.idl``). The resulting 
+generated C++ code will be stored in a folder called ``generated`` in
+the CMake build directory.
+
+Running the Example
+===================
+
+First of all, we need a running |RS| instance with remote administration enabled.
+We will use the |RS| configuration shipped with the example, called 
+``recorder_remote_admin.xml``:
+
+.. code-block:: text
+
+    cd <binary directory>
+    $(CONNEXTDDS_DIR)/bin/rtirecordingservice -cfgFile recorder_remote_admin.xml -cfgName remote_admin
+
+Note: you can run from other directory, just make sure the 
+``recorder_remote_admin.xml`` and ``USER_QOS_PROFILES.xml``
+
+Once the |RS| instance is running, it is ready to receive remote command requests.
+The administration app receives the following command-line parameters:
+
+- The kind of command to be sent, following a *CRUD* pattern (CREATE, READ, 
+  UPDATE, DELETE).
+- The identifier of the resource we're accessing, 
+  e.g. ``recording_services/remote_admin/state``
+- [Optional] The arguments that the command kind needs for performing the action,
+  e.g. ``paused`` (if the objective state of the service is *paused*)
+- [Optional] ``--domain-id``: this parameter should be used when the |RS| 
```

Comment:
In general we don't use the '--param-name' format but '-paramName' instead. It may be ok for this example but we should aim for consistency.

Reply:
@juanlu-rti @fgaranda  yes I can see why you guys like it better ( I do too) but at this point we need to make sure we follow a pattern, which should be also the same for examples. Right now we are probably not consistent in the examples and we should start working towards that. I would make the change if you have a chance if we agree this is the format we should be doing (which is the format of the infsvcs for example).